core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -641,6 +643,16 @@ public SystemSessionProperties(
                         MERGE_PROJECT_WITH_VALUES,
                         "Inline project expressions into values",
                         featuresConfig.isMergeProjectWithValues(),
+                        false),
+                stringProperty(
+                        TIME_ZONE_ID,
+                        "Time Zone Id for the current session",
+                        null,
+                        value -> {
+                            if (value != null) {
+                                getTimeZoneKey(value);
+                            }
+                        },
                         false));
```

Comment:
Mark this property as "hidden". We don't expect users to set it directly.

Reply:
I was actually using this property to check the value set for the time zone.
But you are right, I can get this value through `SELECT CURRENT_TIMEZONE();`